src/editors/containers/ProblemEditor/data/ReactStateOLXParser.js

```diff
@@ -188,6 +188,22 @@ class ReactStateOLXParser {
   addQuestion() {
     const { question } = this.editorObject;
     const questionObject = this.questionParser.parse(question);
+    /* Removes block tags like <p> or <h1> that surround the <label> format.
+      Block tags are required by tinyMCE but have adverse effect on css in studio.
+      */
+    questionObject.forEach((tag, ind) => {
```

Comment:
Will we have to do this in the explanation feild as well for any tags?

Reply:
I don't think so. There is no special formatting for the explanation field.

Reply:
From read the docs, explanation fields has had `<p>` tags baked in in the OLX so I think this issue is more with question and answer fields. 

There's an example of explanation OLX here (from before my RtDoc edits so I'm not just referencing myself):
https://edx.readthedocs.io/projects/edx-partner-course-staff/en/latest/exercises_tools/single_select.html#answer-pools